var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -135,7 +139,7 @@ def install(self, spec, prefix):
 
         # Set up and run tests on installation
         ccompile('-I%s' % prefix.include, '-L%s' % prefix.lib,
-                 '-Wl,-rpath=%s' % (prefix.lib if '+shared' in spec else ''),
+                 '-Wl,-rpath,%s' % (prefix.lib if '+shared' in spec else ''),
```

Comment:
A more robust way to do this would be to use `self.compiler.cc_rpath_arg`. Not every compiler uses `-Wl,-rpath,` (namely the NAG Fortran compiler).

Also, for static builds, this adds `-Wl,-rpath,` to the link line. Maybe it would be better to only add this for shared builds? Or just always add it because it won't hurt.